.github/workflows/rust.yml

```diff
@@ -17,6 +17,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache cargo registry
+      uses: actions/cache@v2.1.7
+      with:
+        path: ~/.cargo/registry
+        key: ${{ runner.os }}-v2-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+
+    - name: Cache cargo index
+      uses: actions/cache@v2.1.7
+      with:
+        path: ~/.cargo/git
+        key: ${{ runner.os }}-v2-cargo-index-${{ hashFiles('**/Cargo.lock') }}
+
+    - name: Cache cargo target dir
+      uses: actions/cache@v2.1.7
+      with:
+        path: contracts/target
```

Comment:
The cache is there but it doesn't seem to get used on `anchor test`, it speeds up linting though.